### ISSUE TITLE
index of a recurrence

### ISSUE THREAD

**Issue by vasugupta09** [OfficeDev/ews-java-api#383]
Hello,

My scenario is if I give MasterRecurring Id to my API, it should return list of all the occurrences inside it with start date, end date and occurrence index number, so that I can further navigate to update , read or delete single occurrence based on its index number using Appointment.bind(service, recurringMaster.Id, indexofOccurences). Is there any way to achieve indexes of all the occurrences? 

Regards,
Vasu Gupta

**Comment by mikebell90** [comment]
No. Occurrences are very irritating in Exchange. From a specific occurrence you can find the master. From the master you can find the first, deleted, modified, exceptions. As far as the index goes, believe it or not, MS just tells you to keep using an index until you get an error. Obviously in anything except well-bounded series, this is nonsensical. But this is a server level issue, not a java library issue.
